### PR TITLE
Add new 16-04 pages

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Add new pages at `/16-04` and `/16-04/gcp`
- Update release schedule chart to have the option of a highlighted row
- Separate the Kernel support schedule chart into it's own file and have the option of showing a particular tab on page load.

**Known bug**: page scrolls to bottom on page load. Trying to fix but ready for design review

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/16-04 and http://0.0.0.0:8001/16-04/gcp
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the pages against [16-04 copy doc](https://docs.google.com/document/d/1wclofqY9b3v_M9igpkYn6or5uXx0B_Xb_nVVzGBbOLY/edit#heading=h.wj8c1jlzgaj3) and [GCP copy doc](https://docs.google.com/document/d/14MeGfBfAiNxnKgJpTXhTEQpKMDAwoNp7Dqh0uqI7Dyc/edit)

## Issue / Card

Fixes [#3690](https://github.com/canonical-web-and-design/web-squad/issues/3690)

### CODE REVIEW

**Comment by carkod** [comment]
Scroll issue seems to be still there. Is the Gantt chart causing it?

File: static/js/src/release-chart.js [hunk 1]
```diff
@@ -144,6 +155,9 @@ function clearCharts() {
   if (document.querySelector("#server-desktop-eol")) {
     document.querySelector("#server-desktop-eol").innerHTML = "";
   }
+  if (document.querySelector("#eol-1604")) {
+    document.querySelector("#eol-1604").innerHTML = "";
+  }
```
Comment (sowasred2012) [inline static/js/src/release-chart.js:160]: This PR didn't introduce this pattern, but there are a lot of examples of querying the document for an ID then, if an element with that is found, querying the document again (unnecessarily) for the same thing. It'd be better to assign the query to a variable so the query is cached, and the document is only queried once (same with other examples in this file):

```suggestion
  const eol1604 = document.querySelector("#eol-1604");
  if (eol1604) {
    eol1604.innerHTML = "";
  }
```

File: static/js/src/tabotronic.js [hunk 2]
```diff
@@ -47,4 +47,13 @@
...
     hashChange();
   });
+
+  const selectedTabGroup = document.querySelector("[data-js-selected-tab]");
+  if (selectedTabGroup) {
+    const selectedTab = selectedTabGroup.querySelector(
+      "#" + selectedTabGroup.dataset.jsSelectedTab
+    );
+    const selectedTabId = selectedTab.getAttribute("aria-controls");
+    tabClickHandler(selectedTab, selectedTabId);
```
Comment (sowasred2012) [inline static/js/src/tabotronic.js:57]: I think this is what's scrolling the page on load - there is an element with `[data-js-selected-tab]` present when you load the page, and `tabClickHandler` scrolls the element into view (line 36).

Comment (sowasred2012) [reply static/js/src/tabotronic.js:57]: An alternative to tabotronic is https://github.com/canonical-web-and-design/canonical.com/blob/master/static/js/tabs.js - it allows switching of tabs without causing the page to jump around.

**Review by sowasred2012 — CHANGES_REQUESTED** [review]

File: static/js/src/chart.js [hunk 3]
```diff
@@ -121,6 +131,19 @@ function emboldenLTSLabels(svg) {
...
+function highlightChartRow(svg, highlightVersion) {
+  svg.selectAll(".tick text").select(function () {
+    var text = this.textContent;
+
+    if (highlightVersion && !text.includes(highlightVersion)) {
+      this.classList.add("chart__label--transparent");
+    }
+    if (text.includes("20") && !text.includes("20.")) {
+      this.classList.remove("chart__label--transparent");
+    }
```
[the remark this answers is not in the document]
Comment (bethcollins92) [reply static/js/src/chart.js:143]: This looks a lot more readable, thanks 👍🏼

File: static/js/src/release-chart.js (hunk 1 above)
Comment (bethcollins92) [reply static/js/src/release-chart.js:160]: Makes sense, I'll update the rest in the file too

**Comment by bethcollins92** [comment]
Made another issue [here](https://github.com/canonical-web-and-design/ubuntu.com/issues/9411) for the 16.04 tab to be the default selected tab on the "Kernel support schedule" table.

File: navigation.yaml [hunk 4]
```diff
@@ -1529,3 +1529,12 @@ gcp:
...
           path: /gcp/thank-you
           hidden: True
+ubuntu1604:
+  title: 16-04
+  path: /16-04
+
+  children:
+    - title: Overview
+      path: /16-04
+    - title: GCP
```
Comment (pmahnke) [inline navigation.yaml:1539]:
```suggestion
    - title: Google Cloud
```

File: templates/16-04/gcp.html [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+{% block meta_description %}Ubuntu 16.04 LTS enters the ESM period in April 2021, with security patches provided for an additional three years beyond the traditional five-year standard support through either an Ubuntu Pro premium image or by attaching a UA subscription.
+{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/14MeGfBfAiNxnKgJpTXhTEQpKMDAwoNp7Dqh0uqI7Dyc/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-bottomed">
+  <div class="row u-vertically-center">
+    <div class="col-6">
+      <h1>Ubuntu 16.04 LTS on Google Cloud</h1>
```
Comment (pmahnke) [inline templates/16-04/gcp.html:13]:
```suggestion
      <h1>Ubuntu 16.04 LTS on Google Cloud (GCP)</h1>
```

File: templates/16-04/gcp.html [hunk 6]
```diff
@@ -0,0 +1,79 @@
+{% extends "16-04/base_16-04.html" %}
+
+{% block title %}Ubuntu 16.04 LTS on GCP{% endblock %}
```
Comment (pmahnke) [inline templates/16-04/gcp.html:3]:
```suggestion
{% block title %}Ubuntu 16.04 LTS on Google Cloud (GCP){% endblock %}
```

**Review by pmahnke — APPROVED** [review]
A few very minor changes + downsize the squirrel!

**Review by sowasred2012 — APPROVED** [review]
LGTM!

**Comment by bethcollins92** [comment]
Thanks @pmahnke and @sowasred2012, made final changes and filed another issue [here](https://github.com/canonical-web-and-design/ubuntu.com/issues/9414) to add the links to the multiple places requested in the [copy doc](https://docs.google.com/document/d/1wclofqY9b3v_M9igpkYn6or5uXx0B_Xb_nVVzGBbOLY/edit?disco=AAAALJrPkOw)

**Pull request merged**